content/updates/2020-11/2-Import-and-export-your-reference-entity-records.md

```diff
@@ -0,0 +1,23 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# Import and export your reference entity records
+::: meta-data type="New" features="Productivity" available="late November"
+
+Having a hard time initializing the records of your reference entities? We got you covered! You can now import an Excel or CSV file containing your records. We also make it possible for you to export them in flat files, once enriched!
```

Comment:
@NolwennP, I wonder if "flat files" speak to our users, for me it's a tech word. Should we use (again) XLSX and CSV file formats or say 
"You can now import and export reference entity records through Excel or CSV."

Reply:
You're right about the "flat files" expression. Not sure, it's well known.
But I cant use your suggestion as exporting is not answering the initialization scenario. I'd suggest:
```suggestion
Having a hard time initializing the records of your reference entities? We got you covered! You can now import an Excel or CSV file containing your records. We also make it possible for you to export them in Excel or CSV, once enriched. So neat!
```